### DISCUSSION TITLE
Error launching source instance

### DISCUSSION THREAD

**Discussion by sousamer** [chgasparoto/curso-aws-com-terraform#79] [Geral]
Bom dia, estou no módulo de variáveis, e na hora do terraform apply -auto-approve aparece o seguinte erro:

│ Error: Error launching source instance: InvalidAMIID.NotFound: The image id '[ami-0056d4296b1120bc3]' does not exist
│       status code: 400, request id: f76c18f6-27ae-4a9f-aab6-aa896e95fb5f
│
│   with aws_instance.web,
│   on main.tf line 11, in resource "aws_instance" "web":
│   11: resource "aws_instance" "web" {

Como conseguiria resolver ?

**Comment by chgasparoto** [comment]
@sousamer qual região você está tentando subir esse EC2 com essa imagem? As AMIs são baseadas na região que você está.

Aqui tem uma lista: https://aws.amazon.com/pt/amazon-linux-ami/
